app/models/exam.py

```diff
@@ -0,0 +1,11 @@
+from app import db
+from datetime import datetime
+
+
+class Exam(db.Model):
```

Comment:
in 1 exam how many question will there be?

Reply:
The exam model I made is just a temporary model to ensure my score endpoints are running. I will change it when the actual Exam model exists. 
I don't know about the number of questions for each exam/material id yet. I will confirm it later in the meet.